src/Tools/ExternalAccess/FSharp/Completion/FSharpCompletionServiceWithProviders.cs

```diff
@@ -17,5 +19,14 @@ internal sealed override CompletionRules GetRules(CompletionOptions options)
             => GetRulesImpl();
 
         internal abstract CompletionRules GetRulesImpl();
+
+        // Disallow implementation since this method is banned in Roslyn
+        public sealed override ImmutableArray<CompletionItem> FilterItems(Document document, ImmutableArray<CompletionItem> items, string filterText)
+#pragma warning disable RS0030 // Do not used banned APIs
+            => base.FilterItems(document, items, filterText);
```

Comment:
should this call FilterItemsDefault?

(and, tbh, i literally do not know what that override is even doing it.  it overrides... but then calls directly into the base, so it would have no effect right?  so why have it?)

Reply:
this is sealed to make sure F# won't accidentally implement the public API in the future (they don't need to override Filteritems today)